pkg/webclient/cloudproviders/cloudproviders.go

```diff
@@ -50,3 +51,61 @@ func (c *CloudProvidersClient) GetCloudProviderByName(ctx context.Context, name
 		KeystoresCount: cp.GetKeystoresCount(),
 	}, nil
 }
+
+func (c *CloudProvidersClient) GetCloudKeystores(ctx context.Context, cloudKeystoreID *string, cloudKeystoreName *string, cloudProviderID *string, cloudProviderName *string) (*domain.CloudKeystore, error) {
```

Comment:
Did you mean `GetCloudKeystore` (no `s`)?

Reply:
Good catch. I was still thinking of previous function that fetched them all, but yeah, in there I'm already reducing the amount to 1

Reply:
Fixed by latest commit